pyquil/operator_estimation.py

```diff
@@ -400,34 +532,46 @@ def group_experiments_clique_removal(experiments: TomographyExperiment) -> Tomog
     return TomographyExperiment(new_cliqs, program=experiments.program, qubits=experiments.qubits)
 
 
-def _validate_all_diagonal_in_tpb(ops: Iterable[PauliTerm]) -> Dict[int, str]:
-    """Each non-identity qubit should result in the same op_str among all operations. Return
-    said mapping.
+def _max_weight_operator(ops: Iterable[PauliTerm]) -> Union[None, PauliTerm]:
```

Comment:
This function as well as `_max_weight_state(..)` are defined slightly later than their first use (in `construct_tpb_graph(..)`); I'd place them immediately before.

Reply:
I've prioritized readability of the diff